docs/blog/2020-community-update.rst

```diff
@@ -17,7 +17,7 @@ The talks were just announced for Prague, and the CFP is open for a little while
 
```

Comment:
CFPs closed last night (14 Sept) and we will be aiming to announce speakers by end of Sept, with the schedule published mid-Oct.

Reply:
Good catch mate